### DISCUSSION TITLE
blog/2024/individual-income-tax/

### DISCUSSION THREAD

**Discussion by giscus** [plantree/press-comment#49] [General]
# blog/2024/individual-income-tax/

小城市普通码农的生活

https://plantree.me/blog/2024/individual-income-tax/

**Comment by thinkingIsMagic** [comment]
感觉可以添加一个文档分类显示 ，比如计算机、经济学、生活等等

**Reply by plantree** [reply]
感谢提醒！有分类，但条目较多，后续会花点时间精简
